src/index.js

```diff
@@ -104,4 +104,7 @@ export default function (moduleOptions) {
   }
   this.options.render.bundleRenderer.directives = this.options.render.bundleRenderer.directives || {}
   this.options.render.bundleRenderer.directives.t = i18nExtensionsDirective
+
+  // Transpile is-https for IE11 support
+  this.nuxt.options.build.transpile.push('is-https')
```

Comment:
Should use `this.options` since that has proper types associated and is shorter.
Thought you will then need to also make sure that `transpile` is an array.

Reply:
Hello,

Which one you'd prefer?

1)
```javascript
this.options.build.transpile = this.options.build.transpile || []
```
2)
```javascript
  if (!this.options.build.transpile) {
    this.options.build.transpile = []
  }
```

Thanks!

Reply:
The first one is fine. Thanks :)